android/src/main/java/com/robinpowered/RNMDMManager/RNMobileDeviceManagerModule.java

```diff
@@ -28,33 +26,27 @@
     public static final String MODULE_NAME = "MobileDeviceManager";
 
     public static final String APP_CONFIG_CHANGED = "react-native-mdm/managedAppConfigDidChange";
-    public static final String APP_LOCK_STATUS_CHANGED = "react-native-mdm/appLockStatusDidChange";
```

Comment:
Leave this in for now -- we want to avoid changing the JS at the moment, and this would add a listener with an `undefined` event name